### PR TITLE
Update `Highlighter` instructions in README 

### PR DESCRIPTION
**Pull request by katjuell**
### What should this PR do?
Resolves [DEVED-221](kjuell/DEVED-221/update-readme-codeblocks) by updating the `README` to have clearer instructions on adding `input` to the `Highlighter` component — specifically around code blocks where you would like spaces to be preserved.

### Why are we making this change?
- The instructions on how to preserve spacing on code `input` for the `Highlighter` component were not as clear as they could be.

### What are the acceptance criteria? 
- The additional explanation and example make sense.

### How should this PR be tested?
- ^^ Check the above
## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,12 +124,22 @@ The component looks like this:
...
     language='Your language here'
     matcher='The code you would like to highlight (optional)'
 />
 ```
+When you have input with line spacing that you would like to preserve, use a template literal for the `input` instead. Also be sure to add an additional blank space to any line where you would like to preserve spacing. For example:
+
+```
+<Highlighter
+    input={`Your code input
+     // Be sure to add a space here!
```
Comment (ltagliaferri) [inline README.md:137]: Can this be `// Be sure to add a space-bar space here!` ? I am not sure if anyone will need this but maybe that makes it clearer....

**Review by ltagliaferri — COMMENTED** [review]
Just one comment about the space space

**Comment by katjuell** [comment]
Going to update this following #339 and the addition of the new component for output blocks.

**Review by ltagliaferri — APPROVED** [review]
Looks good, I modified slightly

**Pull request merged**